### ISSUE TITLE
[selectors-4] Clarify handling keyboard navigation with a cursor for `:focus-visible`

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#3251]
In general `:focus-visible` doesn't show focus when focusing something with a mouse, but does with a keyboard. But actually this isn't the distinction that's needed, the distinction that's needed is whether there's a visible cursor or not. A keyboard navigation mode that moves a cursor around could very reasonably choose to behave like mouse-focusing rather than keyboard-focusing. So we should have an example of that kind of UA choice.
